### DISCUSSION TITLE
Fatal error "Incorrect DB config, or could not connect to DB PDOException: could not find driver

### DISCUSSION THREAD

**Discussion by thehoffm** [YOURLS/YOURLS#3337] [Q&A] [answered]
I've created a quick installation using Softalicious and the initial pages return the following error:

Fatal error
Incorrect DB config, or could not connect to DB
PDOException: could not find driver

I've gone through the initial torubleshooting docs, the config file, .htaccess and can't find anything obvious.

Could someone help walk me through where I might be going wrong?

**Comment by thehoffm** [comment, marked as answer]
To anyone that has a similar error out there, I was missing the php-mysqlnd extension on the server.

**Comment by pblanton** [comment]
In my case the missing extension was nd_pdo_mysql. I went to the PHP version selector on my shared hosting provider's CPanel, and made sure I was using PHP 8.1, then ticked nd_pdo_mysql and that fixed it.
